### DISCUSSION TITLE
Cannot connect to the docker socket. Cannot proceed.

### DISCUSSION THREAD

**Discussion by Obliterous** [nextcloud/all-in-one#4761] [Questions] [answered]
### Steps to reproduce
1. configure reverse proxy
2. install container via simple script

docker run \
--init \
-v /home/dockerp/docks/nextcloud:/mnt/docker-aio-config \
-v /var/run/docker.sock:/var/run/docker.sock:ro \
-p 9090:8080 \
--env APACHE_PORT=11000 \
--env APACHE_IP_BINDING=127.0.0.1 \
--name nextcloud \
--security-opt label=disable \
nextcloud/all-in-one:latest

 

### Expected behavior 

Expected container to start without error

### Actual behavior 

Trying to fix docker.sock permissions internally...
Adding internal www-data to group nobody
Cannot connect to the docker socket. Cannot proceed.
Did you maybe remove group read permissions for the docker socket? AIO needs them in order to access the docker socket.
If SELinux is enabled on your host, see https://github.com/nextcloud/all-in-one#are-there-known-problems-when-selinux-is-enabled
If you are on TrueNas SCALE, see https://github.com/nextcloud/all-in-one#can-i-run-aio-on-truenas-scale

### Host OS 

Linux/ Devuan Daedalus

#### Nextcloud AIO version 

nextcloud/all-in-one:latest

#### Current channel 
------

#### Other valuable info 

No selinux

output from groups & script:

dcon@t5500:~$ groups
dcon users docker
dcon@t5500:~$ ./next-init.sh 
nextcloud
Trying to fix docker.sock permissions internally...
Adding internal www-data to group nobody
Cannot connect to the docker socket. Cannot proceed.
Did you maybe remove group read permissions for the docker socket? AIO needs them in order to access the docker socket.
If SELinux is enabled on your host, see https://github.com/nextcloud/all-in-one#are-there-known-problems-when-selinux-is-enabled
If you are on TrueNas SCALE, see https://github.com/nextcloud/all-in-one#can-i-run-aio-on-truenas-scale
dcon@t5500:~$

**Comment by szaimen** [comment]
Hi, can you for a test disable selinux on the host and check if that makes it work?

**Reply by Obliterous** [reply]
there is no selinux.

**Reply by szaimen** [reply]
Then you need to find out where the docker socket is located on your host and mount the correct one into the container

**Reply by Obliterous** [reply]
root@t5500:/var/run# ls -lhAt docker*
-rw-r--r-- 1 root   root     4 Jun  3 13:43 docker.pid
srw-rw---- 1 root   docker   0 Jun  3 13:43 docker.sock
-rw-r--r-- 1 root   root     5 Jun  3 13:43 docker-ssd.pid

This not it?

**Reply by Zepp10x** [reply]
Having the same problem with a completely vanilla Arch Linux `docker` installation. @Obliterous, have you by any chance found a solution to this problem?

**Comment by szaimen** [comment, marked as answer]
Hm I fear I cannot reproduce this. It works here correctly. How did you install docker? Did you do something special to your docker installation?

**Comment by ricardopereira** [comment]
I'm currently with the same issue. @Obliterous did you found a solution?

**Reply by szaimen** [reply]
Does this help? https://github.com/nextcloud/all-in-one#are-there-known-problems-when-selinux-is-enabled

**Reply by ricardopereira** [reply]
Yes, that helped. Thanks
